### ISSUE TITLE
Fee Collector: Historical per pool summary

### ISSUE THREAD

**Issue by Xeonus** [BalancerMaxis/protocol_fee_allocator#134]
To further enrich the protocol fees page, we shall create a summaries file similar to this one: https://github.com/BalancerMaxis/protocol_fee_allocator/blob/main/fee_allocator/summaries/recon.json

Ideally a json with following information:
1. poolId
2. gaugeId
3. arrays of fees earned, and fees distributed per epoch

In that manner we can plot / visualize per pool / gauge stats that give us insight for historical performance and make it easier for us to put the active fee run into context.

Directly doing API calls / calling the raw run files is inefficient. A summary json file that is created in a similar fashion like the recon.json would be ideal.

**Comment by Tritium-VLK** [comment]
@Xeonus can you maybe provide an example output?

**Comment by Xeonus** [comment]
If we are still interested in per pool performance stats / overall stats over time, I think this is valuable. We could simplify the logic and build automation based off the csvs to create a combined json file? An entry could look something like this:

```
{
  "combined_pool_data": [
  {
    "poolId": "0x...",
    "chain": "mainnet",
    "symbol": "rETH-STABLE",
    "starting_epoch_timestamp": 169xxxx,
    "earned_fees": [123, 123, 123,123],
    "fees_to_vebal": [123, 123, 123,123],
    "fees_to_dao": [123, 123, 123,123],
    "total_incentives": [123, 123, 123,123],
    "aura_incentives": [123, 123, 123,123],
    "bal_incentives": [123, 123, 123,123],
    "redirected_incentives": [123, 123, 123,123],
    "reroute_incentives": [123, 123, 123,123],
  },
  {
   ...
  },
 ]
}
```

**Comment by Tritium-VLK** [comment]
After speaking to @Xeonus and better understanding requirements we agreed to the following.

1: Change the code such that report csvs include the round end date on each line
2: Backfill all historical csvs to have the end date included
3: Create a new github action to concationate all the CSVs together (making sure there is no duplication of pool/chain/end_date)

**Comment by Tritium-VLK** [comment]
Solved by #159
